docs/pages-for-subheaders/rke1-hardening-guide.md

```diff
@@ -243,7 +246,6 @@ services:
   kubelet:
     extra_args:
       feature-gates: RotateKubeletServerCertificate=true
-      protect-kernel-defaults: "true"
```

Comment:
This needs to be inside a specific tab depending on the CIS benchmark version, because it's still needed in CIS 1.23 and CIS 1.24.